libs/features/personalization/personalization.js

```diff
@@ -246,6 +248,26 @@ export function parseConfig(data) {
       variants[vn] = { commands: [] };
     });
 
+    const laterToDate = (laterExpression) => {
+      if (!laterExpression) return null;
+      const regex = /at (\d{1,2}):(\d{1,2}) on the (\d{1,2}) day of (\w+) in (\d{4})/;
+      const matches = laterExpression.match(regex);
+      if (matches && matches.length === 6) {
+        const monthNumber = new Date(Date.parse(`${matches[4]}1, 2012`)).getMonth();
```

Comment:
wouldn't we be better with an array of month string somewhere, or easier, with a format asking for a simpler date format?